src/Nest/XPack/Slm/SnapshotLifecyclePolicyMetadata.cs

```diff
@@ -45,5 +46,36 @@ public class SnapshotLifecyclePolicyMetadata
 		/// </summary>
 		[DataMember(Name = "version")]
 		public int Version { get; internal set; }
+
+		/// <summary>
+		/// If a snapshot is currently in progress this will return information about the snapshot.
+		/// </summary>
+		[DataMember(Name = "in_progress")]
+		public InProgressLifecycleSnapshot InProgress { get; internal set; }
+	}
+
+	/// <summary>
+	/// If a snapshot is in progress when calling the Get Snapshot Lifecycle metadata
+	/// this will hold some minimal information about the in flight snapshot
+	/// </summary>
+	public class InProgressLifecycleSnapshot
```

Comment:
`SnapshotLifecycleInProgress`, to be consistent with other types?

Reply:
++